include/zen/remote.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <znr-remote.h>
+
+struct zn_remote {
+  struct znr_remote *znr_remote;
+
+  struct wl_list peer_list;
```

Comment:
I expect the comment at this line
```suggestion
  struct wl_list peer_list; // zn_peer::link
```

Reply:
Thanks!